src/test/blockfilter_tests.cpp

```diff
@@ -165,4 +170,16 @@ BOOST_AUTO_TEST_CASE(blockfilters_json_test)
     }
 }
 
+BOOST_AUTO_TEST_CASE(blockfilter_type_names)
+{
+    BOOST_CHECK_EQUAL(BlockFilterTypeName(BlockFilterType::BASIC), "basic");
+    BOOST_CHECK_EQUAL(BlockFilterTypeName(static_cast<BlockFilterType>(255)), "");
+
+    BlockFilterType filter_type;
+    BOOST_CHECK(BlockFilterTypeByName("basic", filter_type));
+    BOOST_CHECK_EQUAL(filter_type, BlockFilterType::BASIC);
+
+    BOOST_CHECK(!BlockFilterTypeByName("unknown", filter_type));
+}
+
```

Comment:
14121 is missing changes in https://github.com/bitcoin/bitcoin/pull/14121/files#diff-a5f6e130bdb5dc58520ea6a3266bbd0e4ccca5b9715324514bcf8a4baef2178f

Reply:
@kittywhiskers did you see this?

Reply:
Talked in slack, don't think this is too important. It seems these blocks need to be actual dash blocks, so if someone wants to find interesting blocks to this, then cool, otherwise I don't think it's important